### ISSUE TITLE
Mac OS X: logging stops after starting an iTerm console

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-gui#400]
GNS3 1.3.4, OS: Mac OS X 10.9.5

I newly wanted to use iTerm and configured the predefined iTerm command as a Telnet Console.
Then I dropped a router (cisco 3725 in my example) into the temporary project, started it und started the console.

Everything runs smooth, but in the logging the start of the iTerm console is the last message (except some few QT loggings). This doesn't happen with Terminal as the console program.

```
Jun  4 22:07:24 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:07:24 INFO vm.pyo:99 R1 has started
Jun  4 22:07:27 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:07:27 INFO telnet_console.pyo:90 starting telnet console "osascript -e 'tell application "iTerm"' -e 'activate' -e 'if (count of terminals) = 0 then' -e '  set t to (make new terminal)' -e 'else' -e '  set t to current terminal' -e 'end if' -e 'tell t' -e '  set s to (make new session at the end of sessions)' -e '  tell s' -e '    exec command ("telnet 192.168.1.10 2001")' -e '    delay 1' -e '    repeat while s exists' -e '      delay 1' -e '    end repeat' -e '  end tell' -e 'end tell' -e '
Jun  4 22:07:27 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:07:27 INFO telnet_console.pyo:90 starting telnet console "osascript -e 'tell application "iTerm"' -e 'activate' -e 'if (count of terminals) = 0 then' -e '  set t to (make new terminal)' -e 'else' -e '  set t to current terminal' -e 'end if' -e 'tell t' -e '  set s to (make new session at the end of sessions)' -e '  tell s' -e '    exec command ("telnet 192.168.1.10 2001")' -e '    delay 1' -e '    repeat while s exists' -e '      delay 1' -e '    end repeat' -e '  end tell' -e 'end tell' -e '
Jun  4 22:07:27 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:07:27 INFO telnet_console.pyo:90 starting telnet console "osascript -e 'tell application "iTerm"' -e 'activate' -e 'if (count of terminals) = 0 then' -e '  set t to (make new terminal)' -e 'else' -e '  set t to current terminal' -e 'end if' -e 'tell t' -e '  set s to (make new session at the end of sessions)' -e '  tell s' -e '    exec command ("telnet 192.168.1.10 2001")' -e '    delay 1' -e '    repeat while s exists' -e '      delay 1' -e '    end repeat' -e '  end tell' -e 'end tell' -e '
Jun  4 22:07:41 imac.lan GNS3[2061] <Notice>: QPixmap::scaled: Pixmap is a null pixmap
Jun  4 22:09:33 imac.lan GNS3[2061] <Notice>: QPixmap::scaled: Pixmap is a null pixmap
Jun  4 22:09:40 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:09:40.837 GNS3[2061:507] modalSession has been exited prematurely - check for a reentrant call to endModalSession:
Jun  4 22:09:52 imac.lan GNS3[2061] <Notice>: QPixmap::scaled: Pixmap is a null pixmap
>>> GNS3 finished
Jun  4 22:09:55 imac.lan GNS3[2061] <Notice>: 2015-06-04 22:09:55.199 GNS3[2061:507] modalSession has been exited prematurely - check for a reentrant call to endModalSession:
Jun  4 22:09:55 imac.lan GNS3[2061] <Notice>: Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
Jun  4 22:09:55 imac.lan GNS3[2061] <Notice>: BrokenPipeError: [Errno 32] Broken pipe
```

Even after the stop of the logging I can continue normally, even start new consoles, it's just the logging.

Then I created a start-iTerm script, which holds the long command line. In GNS3 the telnet console command is now "/usr/local/bin/start-iTerm %h %p". That way the logs continue working, even after starting an iTerm console.

 Maybe the problem is the very long command string for iTerm ?

**Comment by noplay** [comment]
I will check

**Comment by noplay** [comment]
Hmm I can not replicate the issue.

The error seem indicate we lost the ouptut somewhere: 
```
BrokenPipeError: [Errno 32] Broken pipe
```

How do you start GNS3? Where are you looking for the log?

**Comment by ehlers** [comment]
I use 2 methods:
- start GNS3 via GUI and look into /var/log/system.log
- start GNS3 via terminal (/Applications/GNS3.app/Contents/MacOS/GNS3) and look at stdout

I just retested, the log in .config/gns3.net/gns3_gui.log does not stop, that continue to log. But as I like to have the log of the server as well, I normally don't use this log.

**Comment by noplay** [comment]
After more test, it's work with 1.4 package and not with 1.3 package. 

It's work from source for both.

I think it's an issue in py2app fixed by the switch to cx_freeze in 1.4.  Also when you run the compiled version on 1.3 the output it not colored but in 1.4 it's ok.
